docs/contributing/Issues.mdx

```diff
@@ -34,6 +34,13 @@ We've found in the past that they result in accidental ["licked cookie"](https:/
 If an issue has been marked as `accepting prs` and an open PR does not exist, feel free to send a PR.
 You don't need to ask for permission.
 
+## Finding Issues
```

Comment:
I find it useful to look through `-linked:pr`, since that's what's available to work on.

However, this has the downside that it filters out issues even if there's a stale/closed closing pr, so maybe we shouldn't recommend that; people can adapt the query to their own needs

Reply:
oh whoops this is already merged 🤦

Reply:
Oop! But yeah I considered the `-linked:pr` but had the same consternation. A lot of old PRs!

I wish there was a way to query to _"no linked PR that is open"_...

Reply:
> I wish there was a way to query to _"no linked PR that is open"_...

+1